### ISSUE TITLE
Basic requirements

### ISSUE THREAD

**Issue by Ingramz** [promod/fastfile-extractor#1]
Idea is simple: have a better tool for doing that than the currently existing ones

Should do a thorough job extracting all sorts of files (or as many as possible).

Batch extraction - all files that are being extracted could be extracted into separate folders or if some file is being overwritten, ask before doing it automatically.

Multiplatform support is not a must, but would be nice to have.

Other: Document file format versions

**Comment by yesitskev** [comment]
Anything to add to this yet?

**Comment by Ingramz** [comment]
No, however if you are interested in developing or have any suggestions - feel free to let me know.

I don't have much time to work on it right now, at most I can provide a working code as a library that would extract contents of one file.

**Comment by yesitskev** [comment]
Ok, if you can dump a current implementation that you have and other resources then I start writing something.

**Comment by yesitskev** [comment]
If you have anything for IWI files also, then dump that also please.  A unified tool (not this project) should also be made for extracting and using / streaming IW assets for whichever reason.
